Locals/frFR.lua

```diff
@@ -0,0 +1,16 @@
+addonName, CraftSim = ...
+
+CraftSim.LOCAL_FR = {}
+
+function CraftSim.LOCAL_FR:GetData()
+	return {
+		-- REQUIRED:
+		[CraftSim.CONST.TEXT.STAT_INSPIRATION] = "Inspiration",
+		[CraftSim.CONST.TEXT.STAT_MULTICRAFT] = "Fabrication multiple",
+		[CraftSim.CONST.TEXT.STAT_RESOURCEFULNESS] = "Ingéniosité",
+		[CraftSim.CONST.TEXT.STAT_CRAFTINGSPEED] = "Vitesse de fabrication",
+		[CraftSim.CONST.TEXT.EQUIP_MATCH_STRING] = "Équipé :",

```

Comment:
Is there really a space between the word and the : ?

Reply:
Yes there is